### PR TITLE
Je/admin reports fixes

### PR DESCRIPTION
**Pull request by emord**
continuation of https://github.com/dimagi/commcare-hq/pull/13186

Some other attributes in admin reports were incorrectly reporting as "no info" for sql domains

bless whoever wrote the initial test for these methods

buddy @proteusvacuum

### CODE REVIEW

File: corehq/ex-submodules/couchforms/tests/test_analytics.py [hunk 1]
```diff
@@ -179,31 +123,93 @@ def test_get_exports_by_form(self):
...
+        cls._60_days = datetime.timedelta(days=60)
+        cls.domain = 'my_crazy_analytics_domain'
+        cls.app_id = uuid.uuid4().hex
+        cls.xmlns = 'my://crazy.xmlns/'
         with trap_extra_setup(ConnectionError):
-            self.elasticsearch = get_es_new()
-            initialize_index_and_mapping(self.elasticsearch, XFORM_INDEX_INFO)
+            cls.elasticsearch = get_es_new()
+            initialize_index_and_mapping(cls.elasticsearch, XFORM_INDEX_INFO)
+            cls.forms = [create_form_and_sync_to_es(cls.now), create_form_and_sync_to_es(cls.now-cls._60_days)]
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/tests/test_analytics.py]: E226 missing whitespace around arithmetic operator

Comment (proteusvacuum) [inline corehq/ex-submodules/couchforms/tests/test_analytics.py]: I think the `create_form_and_sync_to_es` method doesn't return anything so this becomes a list of `None`s. You aren't using the contents of this anywhere, but might be worth changing for whoever comes next.

**Review by proteusvacuum — COMMENTED** [review]
Looks good! Had one comment.

File: corehq/ex-submodules/couchforms/tests/test_analytics.py [hunk 2]
```diff
@@ -179,33 +123,83 @@ def test_get_exports_by_form(self):
...
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/tests/test_analytics.py]: E226 missing whitespace around arithmetic operator

File: corehq/ex-submodules/couchforms/tests/test_analytics.py (hunk 1 above)
Comment (emord) [inline corehq/ex-submodules/couchforms/tests/test_analytics.py]: https://github.com/dimagi/commcare-hq/pull/13304/commits/cc66b621ec7dadc27100a8e5719d60fddbed27f7

Was trying to move around too much code at once

**Review by proteusvacuum — APPROVED** [review]

**Comment by emord** [comment]
Kicked builds. super weird, even the javascript test timed out at 50 min (javascript tests are usually < 10)

**Pull request merged**